### PR TITLE
Injimob 956 vcrenderer ios swift

### PR DESCRIPTION
**Pull request by balachandarg-tw**
Swift Library for Inji VC Renderer to replace placeholders in SVG Template into SVG Image to render in wallet.

### CODE REVIEW

File: Sources/InjiVcRenderer/InjiVcRenderer.swift [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+            return ""
+        }
+        print("values:\n\(values)")
+        
+      
+        do {
+                      let templateURL = "https://5825-2401-4900-1cd1-2813-4f0-c58f-b836-395d.ngrok-free.app/insurance_svg_template.svg" // Replace with your actual URL
+                      let template = try await fetchString(from: templateURL)
+                      print("Content:\n\(template)")
+                      // Assign result to a variable and use it as needed
```
Comment (abhip2565) [inline Sources/InjiVcRenderer/InjiVcRenderer.swift:24]: Comments can we remove if not needed?

File: Package.swift [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    platforms: [
+        .iOS(.v13) // Replace with minimum supported iOS version
+        // Add other platforms if necessary
+    ],
+    products: [
+        // Products define the executables and libraries a package produces, making them visible to other packages.
+        .library(
+            name: "InjiVcRenderer",
+            targets: ["InjiVcRenderer"]),
+    ],
```
Comment (abhip2565) [inline Package.swift:17]: Lets remove comments, If not needed?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,18 @@
...
-Swift library to render VC with SVG template support
+Swift library to render VC with SVG template support. Replaces the placeholders in the SVG Template to generate a valid SVG Image.
+
+
+## Installation
+
+
+To include InjiVcRenderer in your Swift project:
+
+- Clone the InjiVcRenderer library locally.
```
Comment (abhip2565) [inline README.md:10]: Cloning is not mandatory for spm

File: README.md [hunk 4]
```diff
@@ -1,2 +1,18 @@
...
+- Create a new Swift project.
+- Add package dependency: InjiVcRenderer
+
+
+#### API
+
+- `renderSvg(from: vcJsonString)` - expects the Verifiable Credential as parameter and returns the replaced SVG Template.
```
Comment (abhip2565) [inline README.md:17]: I see we are exposing only one api, can we make other functions, `internal` or `private`

File: Tests/InjiVcRendererTests/InjiVcRendererTests.swift [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+        renderer.renderSvg(from: jsonString) { output in
+            result = output
+            expectation.fulfill()
+        }
+
+        waitForExpectations(timeout: 1.0, handler: nil)
+        XCTAssertNil(result)
+    }
+
+    // Test date formatting
```
Comment (abhip2565) [inline Tests/InjiVcRendererTests/InjiVcRendererTests.swift:130]: Comments can be removed

**Review by abhip2565 — APPROVED** [review]

File: Sources/InjiVcRenderer/InjiVcRenderer.swift (hunk 1 above)
Comment (balachandarg-tw) [reply Sources/InjiVcRenderer/InjiVcRenderer.swift]: Done

File: README.md (hunk 3 above)
Comment (balachandarg-tw) [reply README.md]: Updated

File: README.md (hunk 4 above)
Comment (balachandarg-tw) [reply README.md:16]: Done

File: Tests/InjiVcRendererTests/InjiVcRendererTests.swift (hunk 5 above)
Comment (balachandarg-tw) [reply Tests/InjiVcRendererTests/InjiVcRendererTests.swift]: Done

**Review by abhip2565 — APPROVED** [review]

**Review by swatigoel — APPROVED** [review]

**Pull request merged**